fs/xfstests.py.data/xfstests.yaml

```diff
@@ -1,4 +1,19 @@
 setup:
-    runargs : !mux
-        skip_dangerous: True
-        test_range: '2,12,34-87,91,99-112'
+    skip_dangerous: True
+    file_system: 'ext4'
```

Comment:
@harish-24 did not notice this first. Can you please change this parameter name as `fs` to be uniform across all tests.